site-config/lungmap/dev/index/samplesEntityConfig.ts

```diff
@@ -0,0 +1,211 @@
+        header: HCA_DCP_CATEGORY_LABEL.PAIRED_END,
+        id: HCA_DCP_CATEGORY_KEY.PAIRED_END,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedProtocolWorkflow,
+        } as ComponentConfig<typeof C.NTagCell, SamplesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.WORKFLOW,
+        id: HCA_DCP_CATEGORY_KEY.WORKFLOW,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorOrganismAge,
+        } as ComponentConfig<typeof C.NTagCell, SamplesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.ORGANISM_AGE,
+        id: HCA_DCP_CATEGORY_KEY.ORGANISM_AGE,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildAggregatedDonorBiologicalSex,
+        } as ComponentConfig<typeof C.BasicCell, SamplesResponse>,
+        header: "Sex",
+        id: HCA_DCP_CATEGORY_KEY.BIOLOGICAL_SEX,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedSpecimenDisease,
+        } as ComponentConfig<typeof C.NTagCell, SamplesResponse>,
+        header: "Disease (Specimen)",
+        id: HCA_DCP_CATEGORY_KEY.SPECIMEN_DISEASE,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorDisease,
+        } as ComponentConfig<typeof C.NTagCell, SamplesResponse>,
+        header: "Disease (Donor)", // TODO confirm header
+        id: HCA_DCP_CATEGORY_KEY.DONOR_DISEASE,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorDevelopmentStage,
+        } as ComponentConfig<typeof C.NTagCell, SamplesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.DEVELOPMENT_STAGE,
+        id: HCA_DCP_CATEGORY_KEY.DEVELOPMENT_STAGE,
+        width: { max: "1fr", min: "148px" },
+      },
+      {
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildTotalCells,
+        } as ComponentConfig<typeof C.BasicCell, SamplesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.CELL_COUNT,
+        id: HCA_DCP_CATEGORY_KEY.CELL_COUNT,
+        width: { max: "1fr", min: "124px" },
+      },
+    ],
+    defaultSort: {
+      desc: SORT_DIRECTION.ASCENDING,
+      id: HCA_DCP_CATEGORY_KEY.SAMPLE_ID,
+    },
+  } as ListConfig<SamplesResponse>,
+  listView: {
+    subTitleHero: hcaSubTitleHero,
+  },
```

Comment:
Hi @jpaten, would you mind removing the `listView` from `samplesEntityConfig` lines `207-209`.